From ecbb859d88eacad10bf556044abaeaed6c9b32fc Mon Sep 17 00:00:00 2001
From: "berto@igalia.com"
 <berto@igalia.com@268f45cc-cd09-0410-ab3c-d52691b4dbfc>
Date: Thu, 2 Nov 2017 13:41:16 +0000
Subject: [PATCH] Unreviewed, fix typos in library names for the GTK+ port.

* Source/cmake/OptionsGTK.cmake:

git-svn-id: http://svn.webkit.org/repository/webkit/trunk@224329 268f45cc-cd09-0410-ab3c-d52691b4dbfc
---
 ChangeLog                     | 6 ++++++
 Source/cmake/OptionsGTK.cmake | 4 ++--
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/Source/cmake/OptionsGTK.cmake b/Source/cmake/OptionsGTK.cmake
index 2b440a27376..6c99868ae86 100644
--- a/Source/cmake/OptionsGTK.cmake
+++ b/Source/cmake/OptionsGTK.cmake
@@ -367,11 +367,11 @@ endif ()
 if (USE_WOFF2)
     find_package(BrotliDec 1.0.1)
     if (NOT BROTLIDEC_FOUND)
-       message(FATAL_ERROR "librotlidec is needed for USE_WOFF2.")
+       message(FATAL_ERROR "libbrotlidec is needed for USE_WOFF2.")
     endif ()
     find_package(WOFF2Dec 1.0.1)
     if (NOT WOFF2DEC_FOUND)
-       message(FATAL_ERROR "liwoff2dec is needed for USE_WOFF2.")
+       message(FATAL_ERROR "libwoff2dec is needed for USE_WOFF2.")
     endif ()
 endif ()
 
-- 
2.16.2

